eva/configuration/configuration_manager.py

```diff
@@ -28,9 +30,37 @@ def __new__(cls):
         if cls._instance is None:
             cls._instance = super(ConfigurationManager, cls).__new__(cls)
 
-            with open(os.path.join(EVA_DIR, "eva.yml"), 'r') as ymlfile:
+            ymlpath = None
+            if importlib_resources.is_resource('eva', 'eva.yml'):
+                with importlib_resources.path('eva', 'eva.yml')as path:
+                    ymlpath = path
+            else:  # For local dev environments without package installed
+                ymlpath = os.path.join(EVA_INSTALLATION_DIR, "eva.yml")
+
+            with open(ymlpath, 'r') as ymlfile:
                 cls._cfg = yaml.load(ymlfile, Loader=yaml.FullLoader)
 
+            dataset_location = cls._instance.get_value("core", "location")
+            database_uri = cls._instance.get_value("core",
+                                                   "sqlalchemy_database_uri")
+            if not dataset_location or not database_uri:
```

Comment:
If they are defined in the config file, we might still need to check if the directory already exists or not?